modules/installation-overview.adoc

```diff
@@ -7,50 +7,148 @@
+
+The install configuration files are all pruned when you run the installation program,
+so be sure to back up all configuration files that you want to use again.
+
+[IMPORTANT]
+====
+You cannot modify the parameters that you set during installation, but you can
+modify many cluster attributes after installation.
+====
+
+
+[discrete]
+=== IPI cluster installation process
+
+[IMPORTANT]
+====
+In version {product-version}, you can perform an
+Installer Provisioned Infrastructure (IPI) install {product-title} on only
 Amazon Web Services (AWS).
-By default, the installation program acts as an installation wizard, prompting you
+====
+
+The default installation type is IPI on AWS. By default, the installation
+program acts as an installation wizard, prompting you
 for values that it cannot determine on its own and providing reasonable default
 values for the remaining parameters. You can also customize the installation
 process to support advanced infrastructure scenarios. The installation program provisions
 the underlying infrastructure for the cluster.
 
-////
-In supported cloud environments, the installation program can also provision the underlying
-infrastructure for the cluster. If possible, use this feature to avoid
+You can install either a standard cluster or a customized cluster. With a
+standard cluster, you provide minimum details that you need to install the
+cluster. With a customized cluster, you can specify more details about the
+platform, such as the number of machines that the control plane uses, the type
+of virtual machine that the cluster deploys, or the CIDR range for the
+Kubernetes service network.
+
+If possible, use this feature to avoid
 having to provision and maintain the cluster infrastructure. In all other
 environments, you use the installation program to generate the assets that you need to
 provision your cluster infrastructure.
-////
-
-[NOTE]
-====
-In {product-title} version 4.0, the installation program can only provision infrastructure
-for clusters on AWS.
-//If you use another cloud provider or on-premises hardware, you must provide the infrastructure.
-====
-
-[discrete]
-== Cluster installation process
 
-In supported cloud environment scenarios, {product-title} manages all aspects of
+With IPI clusters, {product-title} manages all aspects of
 the cluster, including the operating system itself. Each machine boots with a
 configuration that references resources hosted in the cluster that it joins.
 This configuration allows the cluster to manage itself as updates are applied.
 
+[discrete]
+=== UPI cluster installation process
+
+You can also install {product-title} on infrastructure that you provide. You use the
+installation program to generate the assets that you need to provision the
+cluster infrastructure, create the cluster infrastructure, and then deploy
+the cluster to the infrastructure that you provided.
+
+With UPI clusters, you must manage and maintain the cluster nodes yourself.
```

Comment:
"nodes" -> "resources", since you're on the hook for subnets and load balancers and route tables, etc., etc.  Maybe list a few explicitly here?  We want this to sound as unappealing and tedious as it's going to be so folks are more likely to make their lives easier by taking the IPI route ;).